src/components/Insight/InsightsScrollable.module.scss

```diff
@@ -1,5 +1,5 @@
 @import '~@santiment-network/ui/variables.scss';
-@import '../../mixins.scss';
+@import '../../mixins';
```

Comment:
Is it possible here to use `san-ui` mixins ?

Reply:
of course)
missed that moment, when in `san-ui` appeared `responsive` mixin (: